apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.utils.ts

```diff
@@ -126,7 +127,7 @@ export const generateUpdateColumnPayload = (
   const type = field.isArray ? `${field.format}[]` : field.format
   const comment = ((field.comment?.length ?? '') === 0 ? '' : field.comment)?.trim()
 
-  const check = field.check?.trim()
+  const check = field.check?.trim() ? field.check?.trim() : undefined
```

Comment:
what is this supposed to do?

Reply:
Based on https://github.com/supabase/supabase/blob/master/packages/pg-meta/src/pg-meta-columns.ts#L341, we should set empty string to null.

```
const check = field.check?.trim() || null
```